src/freertos/posix/config/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+set(LIB_NAME "stub_freertos_posix")
+set(LIB_CONFIG_NAME "stub_freertos_posix_config")
```

Comment:
It's not really a stub anymore since it's the whole freertos lib

Reply:
Done